html/interaction/focus/processing-model/preventScroll-nested-scroll-elements.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<title>focus(options) - preventScroll on nested scroll elements</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+  #scrollBox { width: 400px; height: 300px; border: 1px solid }
+  .bigbox { width: 380px; height: 300px; border: 1px solid }
+  .box { width: 380px; height: 150px; border: 1px solid }
+</style>
+<div id="scrollBox" style="overflow-y: scroll;">
```

Comment:
Minor opinion, but how about specifying `scrollBox` for the class instead of id?

Reply:
Sure, that was a bit inconsistent. I also made some other minor fixes.

Reply:
Thank you!